### DISCUSSION TITLE
JSON: Syntax error, malformed JSON.

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#630] [Q&A]
When updating the data from one of our Districts we just started receiving the above message.
Any assistance would be greatly appreciated.

Scott F.
Area 59 Web Servant

**Comment by brianw-area24** [comment]
In order to look into this farther, we are going to need to know the feed you're trying to import that's giving this error, either by posting the URL here, or sending it to devs@code4recovery.org

This is similar to another question we recently received, see Discussion #616
